### PR TITLE
Add generation tests. Fix env in proto. 

### PR DESCRIPTION
**Pull request by mhirna**
Issue #9, adding tests for generation.

### CODE REVIEW

File: graph_generation/test_attribute_generator.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        dataset_params = DatasetParams(0, {})
+        system_params = SystemParams(0, {}, {})
+        dataset_to_system_params = DatasetToSystemParams({}, {}, {}, {})
+        collection_params = CollectionParams({}, {}, 0, {}, 0, {}, 0)
+        data_integrity_params = DataIntegrityParams({})
+        processing_params = ProcessingParams({}, {})
+        connection_params = ConnectionParams(0, 0)
+        return collection_params, dataset_params, system_params, data_integrity_params, processing_params, connection_params
+
+    def test__generate_time(self):
```
Comment (kboyeldieu) [inline graph_generation/test_attribute_generator.py:33]: I'm not a big fan of these double underscores in method names

File: graph_generation/test_connection_generator.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+            list_sum -= curr_element
+
+        # Fix the sum of the list if last element was too large.
+        random_array[random_array.index(max(random_array))] += list_sum
+        return random_array
+
+    def test_get_one_to_many_connections(self):
+        """
+        Tests if method get_one_to_many_connection generates connections with given distribution.
+        To compare config parameter distribuion, and generated distribution Kolmogorov-Smirnov statistic is used.
```
Comment (kboyeldieu) [inline graph_generation/test_connection_generator.py]: nit: reword in "Kolmogorov-Smirnov statistic is used to compare config parameter and generated distributions"?

**Review by kboyeldieu — APPROVED** [review]
Looks good! Thanks for adding tests. Please address the couple nits before submitting

File: graph_generation/test_attribute_generator.py (hunk 1 above)
Comment (mhirna) [reply graph_generation/test_attribute_generator.py:33]: This is a standard in Python unit tests to name test methods "test_" + name of method, and as these methods are private, they start with "_", like "_generate_time", so we get the double underscore in the end.

File: graph_generation/test_connection_generator.py (hunk 2 above)
Comment (mhirna) [reply graph_generation/test_connection_generator.py]: Okey, can rephrase

**Pull request merged**